WcaOnRails/config/locales/zh-CN.yml

```diff
@@ -547,12 +566,12 @@ zh-CN:
         #original_hash: da39a3e
         competition_id: ''
       user:
-        #original_hash: 87e1e6f
+        #original_hash: 7588d71
         dob: 按“YYYY-MM-DD"格式输入你的生日。
-        #original_hash: d0bb74a
+        #original_hash: ac908c9
         name: >-
           正确输入你的全名，比如<strong>Baiqiang Dong (董百强)</strong>。别偷懒只输入<strong>Baiqiang
-          (百强)</strong>
+          (百强)
```

Comment:
this `</strong>` is needed for the second `<strong>` !

Reply:
I will change it later

Reply:
https://github.com/thewca/worldcubeassociation.org/pull/7609